src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -35,7 +36,30 @@ public class RestoreCommand
 
         public Guid ParentId { get; }
 
-        public const string ProjectRestoreInformation = "ProjectRestoreInformation";
+        // names for ProjectRestoreInformation and intervals
+        private const string ProjectRestoreInformation = "ProjectRestoreInformation";
+        private const string ErrorCodes = "ErrorCodes";
+        private const string ErrorMessages = "ErrorMessages";
+        private const string WarningCodes = "WarningCodes";
+        private const string WarningMessages = "WarningMessages";
+        private const string RestoreSuccess = "RestoreSuccess";
```

Comment:
`RestoreSuccess` is the status for current project restore? or something else?

Reply:
https://github.com/NuGet/NuGet.Client/pull/2169/files#diff-ae3ea71e74915c8f0634a376599d6b87R269